src/tasks/task1.js

```diff
@@ -0,0 +1,28 @@
+let _chunk;
+const _reverse = str => {
+  return str.split('').reverse((a, b) => {
+    if (a > b) return 0;
+    if (a < b) return -1;
+  }).join(',').replace(/,/g, '')
```

Comment:
Can you join with empty space, so you dont need replace it later?

Reply:
Done.